packages/core/src/main/logger/close-ipc-logging-listener.injectable.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import ipcFileLoggerInjectable from "./ipc-file-logger.injectable";
+import { getMessageChannelListenerInjectable } from "../../common/utils/channel/message-channel-listener-injection-token";
+import {
+  closeIpcFileLoggerChannel,
+} from "../../common/logger/ipc-file-logger-channel";
+
+const closeIpcFileLoggingListenerInjectable = getMessageChannelListenerInjectable({
+  id: "close-ipc-file-logging",
+  channel: closeIpcFileLoggerChannel,
+  handler: (di) => (fileId) =>
+    di
+      .inject(ipcFileLoggerInjectable)
+      .close(fileId),
```

Comment:
We should inject `ipcFileLogger` once and then use it afterwards

```suggestion
  handler: (di) => {
    const ipcFileLogger = di.in.inject(ipcFileLoggerInjectable);
    
    return (fileId) => ipcFileLogger.close(fileId);
  },
```

Reply:
Thanks!